language/move-prover/doc/user/spec-lang.md

```diff
@@ -1039,83 +1089,29 @@ struct S { x: u64, y: u64 }
 fun x_or_y(b: bool, s: &mut S): &mut u64 {
     if (b) &mut s.x else &mut s.y
 }
-spec fun x_or_y {
+spec x_or_y {
     ensures b ==> result == s.x;
     ensures !b ==> result == s.y;
 }
 ```
 
-We are not able to specify the *full* semantics of `x_or_y` in MSL, because we cannot capture the semantics
-of mutable references. While we can say something about the value behind the reference at function exit, subsequent
+We are not able to specify the *full* semantics of `x_or_y` in MSL, because we cannot capture the semantics of
+mutable references. While we can say something about the value behind the reference at function exit, subsequent
 effects as in `*x_or_y(b, &mut s) = 2` cannot be specified.
 
 However, the Move prover *does* understand the meaning of such functions -- the restriction is only in what we
-can specify. Practically this means we cannot make the function `x_or_y` opaque, and must let verification rely on that
-the prover directly works with the implementation. Specifically, we can verify the following (opaque):
+can specify. Practically this means we cannot make the function `x_or_y` opaque, and must let verification rely
+on that the prover directly works with the implementation. Specifically, we can verify the following (
+opaque):
```

Comment:
This will cause an extra space in front of the word `opaque`.

```suggestion
on that the prover directly works with the implementation. Specifically, we can verify the following
(which can then be opaque):
```